ele/element.py

```diff
@@ -250,8 +250,10 @@ def infer_element_from_string(string):
 elements = []
 with open(JSON_PATH) as json_file:
     elements_dict = json.load(json_file)
+    elements_dict = {int(key): value for key, value in elements_dict.items()}
```

Comment:
Very Nitpicky, but a suggestion nonetheless.

```suggestion

    elements_dict = json.load(json_file, object_hook=lambda d: {int(k) if k.lstrip('-').isdigit() else k: v for k, v in d.items()})
```

Reply:
No worries. Can you explain what you're doing there?

Reply:
The `json.load` function accepts  few keyword arguments (`object_hook`/ `object_pairs_hook`) that can be used for key/value transformation while performing the load. Since, this is almost what we want here, but the dictionary is recursive (dict of dict). So, if a transform is possible (cases where a string key can be transformed to an integer (the `isdigit` check in lambda), we are instructing the loader to do so.

Reply:
Per our discussion I'm going to leave this code as is. I appreciate the suggestion though :).